app/views/sign_up/select_email/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<% self.title = t('titles.select_email') %>
+
+<%= render StatusPageComponent.new(status: :info, icon: :question) do |c| %>
+  <% c.with_header { t('titles.select_email') } %>
+  <p class="margin-top-4">
+    <%= I18n.t('help_text.select_preferred_email', sp: @sp_name, app_name: APP_NAME) %>
+  </p>
+
+  <%= simple_form_for('', url: sign_up_select_email_path) do |f| %>
+    <% @user_emails.each do |email, index| %>
+      <fieldset class="usa-fieldset">
+        <div class="usa-radio">
```

Comment:
Did you try using the [radio button builder](https://github.com/18F/identity-idp/blob/b8a3933bc536590a84a0e1c459445a4b9ee0f71c/config/initializers/simple_form.rb#L82-L115) like we [use with language selection for account creation](https://github.com/18F/identity-idp/blob/main/app/views/shared/_email_languages.html.erb)? One thing I notice in particular is that the radio buttons aren't expected to extend full-width of the screen, which the builder helps manage.

Reply:
I'm not sure if it was because of the different way I have the form or something else but I wasn't able to get the buttons to work here like they're used in the email selection. I could revisit if you think it might be better?

Reply:
I think the primary concern at this point would be getting the width correct if possible.